src/config.rs

```diff
@@ -497,11 +517,16 @@ fn print_panic_info(printer: &PanicPrinter<'_>, out: &mut fmt::Formatter<'_>) ->
         write!(&mut separated.ready(), "{}", env_section)?;
     }
 
+    if let Some(ref section) = printer.section {
+        write!(&mut separated.ready(), "{}", section)?;
+    }
```

Comment:
Do you prefer having this print at the very end of the report? I was thinking it might be better to put this between the panic message and the span trace, or perhaps right before the env section, but I don't have a really strong preference so I'll let you pick where it goes based on your preferred use case.

Reply:
I don't prefer it, it was a accident while rebasing :sweat_smile: